app/javascript/gobierto_data/webapp/components/sets/data/SQLEditorResults.vue

```diff
@@ -185,7 +190,9 @@ export default {
     onSaveEventHandler(opts) {
       // get children configuration
       const config = this.$refs.viewer.getConfig()
-      this.$root.$emit("storeCurrentVisualization", config, opts);
+      if (!this.registrationDisabled) {
```

Comment:
What's the deal about excluding the `config` variable out of the conditional block? If you only use it inside the block, move it inside the block